src/V3AstNodes.h

```diff
@@ -1686,6 +1749,44 @@ class AstAssocSel final : public AstNodeSel {
     virtual int instrCount() const override { return widthInstrs(); }
 };
 
+class AstWildcardSel final : public AstNodeSel {
+    // Parents: math|stmt
+    // Children: varref|arraysel, math
+private:
+    void init(AstNode* fromp) {
+        if (fromp && VN_IS(fromp->dtypep()->skipRefp(), WildcardArrayDType)) {
+            // Strip off array to find what array references
+            dtypeFrom(VN_AS(fromp->dtypep()->skipRefp(), WildcardArrayDType)->subDTypep());
+        }
+    }
+
+public:
+    AstWildcardSel(FileLine* fl, AstNode* fromp, AstNode* bitp)
+        : ASTGEN_SUPER_WildcardSel(fl, fromp, bitp) {
+        init(fromp);
+    }
+    ASTNODE_NODE_FUNCS(WildcardSel)
+    virtual AstNode* cloneType(AstNode* lhsp, AstNode* rhsp) override {
+        return new AstWildcardSel(this->fileline(), lhsp, rhsp);
```

Comment:
```suggestion
        return new AstWildcardSel{this->fileline(), lhsp, rhsp};
```

In your entire patch set use {} for all "new" you are adding/editing.